Cargo.toml

```diff
@@ -32,8 +32,9 @@ name = "transaction_tests"
 harness = false
 
 [dev-dependencies]
+failure = "0.1"
+futures = "*"
```

Comment:
Better to pin this to specific version (i.e. `0.1`) as Clarity doesn't use Cargo.lock to ensure accidental updates doesn't happen. Also interesting read: https://doc.rust-lang.org/cargo/faq.html#can-libraries-use--as-a-version-for-their-dependencies. Not sure if crates.io rejects dev-deps with `*` too, but still antipattern IMO.

Reply:
Yup, it's far less ready than I thought :P